source/fileUtils.py

```diff
@@ -0,0 +1,24 @@
+#fileUtils.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2006-2016 NV Access Limited
```

Comment:
This should instead be:

`+#Copyright (C) 2017 NV Access Limited, Bram Duvigneau`